doc/api/events.md

```diff
@@ -167,6 +167,9 @@ myEmitter.emit('error', new Error('whoops!'));
 ```
 
 ## Class: EventEmitter
+<!-- YAML
+added: v0.1.90
```

Comment:
This should be [v0.1.33](https://github.com/nodejs/node/commit/6befc72f829bfc7c027da8e96b12cf9d2a8f6a8e).